base_build/scripts/locale-filter.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+## Filter list of locales from a given filter args
+## Parse into array
+LANG_ARR=(${LANGS//,/ })
+echo "" > /etc/locale.gen
+for i in "${LANG_ARR[@]}"; do
+  cat /etc/all.locale.gen | grep "$i" >> /etc/locale.gen
+done
```

Comment:
This is a helper script to filter desired locale in the build stage of postgis-base.
The scripts is unused if we build all locale

Reply:
Long overdue. Been a pending issue for people building locally a d Travis tests